### DISCUSSION TITLE
Json Viewer Component

### DISCUSSION THREAD

**Discussion by viniciusfaleiro** [ToolJet/ToolJet#7990] [Q&A]
Is there any json viewer component on tolljet or a way to do so by adding a custom component? I'm not really familiar with react and was not able to use react-json-view on my page.

**Comment by ShazanRizvi** [comment]
Hey @viniciusfaleiro 
Currently, there is no component for the JSON viewer but you can try using custom component for making a JSON viewer. I would recommend to go through our docs of the custom component: https://docs.tooljet.com/docs/widgets/custom-component/

You can also join our slack community to get help from our team and our community members. Here is the invite: https://tooljet.com/slack.
